client/app/scripts/superdesk-archive/controllers/list.js

```diff
@@ -5,13 +5,14 @@ define([
     'use strict';
 
     ArchiveListController.$inject = [
-        '$scope', '$injector', '$location',
+        '$scope', '$injector', '$location', '$timeout',
         'superdesk', 'session', 'api', 'desks', 'ContentCtrl', 'StagesCtrl'
     ];
-    function ArchiveListController($scope, $injector, $location, superdesk, session, api, desks, ContentCtrl, StagesCtrl) {
+    function ArchiveListController($scope, $injector, $location, $timeout, superdesk, session, api, desks, ContentCtrl, StagesCtrl) {
 
-        var resource;
-        var self = this;
+        var resource,
+            timeout,
+            self = this;
 
         $injector.invoke(BaseListController, this, {$scope: $scope});
```

Comment:
Well, sometimes it's needed because `this` can have different meanings depending on the context, thus storing a reference to the original `this` is fine (it doesn't have to be called "self", though).

But if the reference is not really needed in this controller, then it's best to simply remove the variable?